src/routes/mod_creation.rs

```diff
@@ -0,0 +1,228 @@
+use crate::file_hosting::{upload_file, AuthorizationData, UploadUrlData};
+use crate::models::ids::random_base62;
+use crate::models::mods::{
+    FileHash, GameVersion, Mod, ModId, Version, VersionFile, VersionId, VersionType,
+};
+use crate::models::teams::{Team, TeamId, TeamMember};
+use actix_multipart::{Field, Multipart};
+use actix_web::web::Data;
+use actix_web::{middleware, post, HttpResponse};
+use bson::Bson;
+use chrono::Utc;
+use futures::StreamExt;
+use mongodb::Client;
```

Comment:
The models module should **not** include database stuff. This module is only for API returned values, present in the body.

Reply:
Yep, like I said in the todo list, I'll be moving them into the database models.